src/generated/resources/schema.graphql

```diff
@@ -50,6 +51,17 @@ type CCloudSchemaRegistry {
   uri: String!
 }
 
+type CcloudFlinkComputePool {
+  cloud: String
```

Comment:
has been mapped to "provider" in the Java files so this query works:

```
❯ curl -s -X POST "http://localhost:26636/gateway/v1/graphql" \
  -H "Content-Type: application/json" \
  -d '{
    "query": "query { ccloudConnectionById(id: \"c1\") { environments { flinkComputePools { id display_name provider region max_cfu } } } }"
  }' | jq .
{
  "data": {
    "ccloudConnectionById": {
      "environments": [
        {
          "flinkComputePools": [
            {
              "id": "lfcp-blahblah",
              "display_name": "AWS.us-east-2.env-blahblah",
              "provider": "AWS",
              "region": "us-east-2",
              "max_cfu": 10
            },
etc
```

Reply:
Can you re-generate this file by, for instance, starting the application in dev mode? It should include "provider" not "cloud" here.